### PR TITLE
Add page refresh to delete stream

### PR DESCRIPTION
**Pull request by gdzooks**
# General info

**Issue number**: N/A

**Task description**: 
 - Added refresh (redirect to same page) after stream delete
 
 

# Testing
**Test coverage**:
- 100% :tada:

# Additional notes

**Note to reviewers**:
- @FGRCL messaged me about 6882b15 so i made the change here, since this is a small PR anyway xD

**To do before merging**:
- n/a

### CODE REVIEW

File: frontend/src/api/DeviceApi.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ export function getReceivers(callback) {
 
 export function deleteDevice(deviceId, callback) {
   axios
-    .delete(`REACT_APP_DEVICE=$DOMAIN/device/${deviceId}`)
+    .delete(`${process.env.REACT_APP_DEVICE}/${deviceId}`)
```
Comment (FGRCL) [inline frontend/src/api/DeviceApi.js]: Yes

File: frontend/src/streamlist/DeleteStream.jsx [hunk 2]
```diff
@@ -22,6 +25,8 @@ export default function DeleteStream(props) {
   };
   const confirmDelete = () => {
     StreamApi.deleteStream(deleteId);
+    history.push("/Streaming");
+    history.go(0);
```
Comment (sharon-ho) [inline frontend/src/streamlist/DeleteStream.jsx]: Glad `.go(0)` was added after some discussion otherwise the page wouldn't refresh!

**Review by sharon-ho — DISMISSED** [review]
LGTM!

**Review by FGRCL — DISMISSED** [review]
I tested it and it works well on my end. I just want to double check something. When I open delete a device, it throws me back to the device list page, but I can still see the device I just deleted until I refresh the page. That's expected behavior right?

**Comment by gdzooks** [comment]
> I tested it and it works well on my end. I just want to double check something. When I open delete a device, it throws me back to the device list page, but I can still see the device I just deleted until I refresh the page. That's expected behavior right?

uhh you shouldn't have to refresh the page again because `history.go(0)` is already a page refresh, not sure why that's happening. could this be a delay issue with the device actually getting deleted in the backend?

**Comment by felixlapierre** [comment]
This feature isn't working for me on my end :/

**Review by felixlapierre — CHANGES_REQUESTED** [review]
This feature wasn't working on my end since the page refreshes before the backend request can go through. I created https://github.com/bean-pod/switchboard/pull/294 with the hotfix, which should be merged into this before we merge the feature to master. I'm fixing the failing tests on the hotfix right now.

**Review by felixlapierre — APPROVED** [review]
My changes were applied LGTM

**Review by sharon-ho — APPROVED** [review]

**Pull request merged**